firoptions/FiroptionConfiguration/FiroptionConfiguration/AppDelegate.swift

```diff
@@ -45,12 +45,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // Note: this one is not deleted, so is the default below.
     // [START default_configure_vars]
     // Configure with manual options.
-    let secondaryOptions = FirebaseOptions(googleAppID: "1:27992087142:ios:2a4732a34787067a", gcmSenderID: "27992087142")
+    let secondaryOptions = FirebaseOptions(googleAppID: "1:27992087142:ios:2a4732a34787067a",
+                                           gcmSenderID: "27992087142")
     secondaryOptions.bundleID = "com.google.firebase.devrel.FiroptionConfiguration"
     secondaryOptions.apiKey = "AIzaSyBicqfAZPvMgC7NZkjayUEsrepxuXzZDsk"
+    secondaryOptions.trackingID = "UA-12345678-1"
     secondaryOptions.clientID = "27992087142-ola6qe637ulk8780vl8mo5vogegkm23n.apps.googleusercontent.com"
     secondaryOptions.databaseURL = "https://myproject.firebaseio.com"
     secondaryOptions.storageBucket = "myproject.appspot.com"
+    secondaryOptions.projectID = "projectid-12345"
```

Comment:
I am cool with this (the FIS team will be happy) but FWIW on the Android snippets I went the opposite direction.  I only showed the 3 required params, since the real breaking change here is that they're actually required now.

So if you want to leave all of these, add a code comment emphasizing the main 3.

Reply:
Done.